### ISSUE TITLE
Pathfinder Version Checking via RPC?

### ISSUE THREAD

**Issue by DaveWK** [eqlabs/pathfinder#1193]
Is there a REST endpoint that displays the current running version of pathfinder (IE 0.64, 0.60, etc..) ? I didn't see any info on this in the docs, but this is very helpful for health/maintenance

**Comment by Mirko-von-Leipzig** [comment]
We have a pathfinder specific endpoint [`pathfinder_version`](https://github.com/eqlabs/pathfinder/blob/main/doc/rpc/pathfinder_rpc_api.json#L9-L21)

**Comment by tabaktoni** [comment]
I also tried this endpoint unusefully v.0.6.6
Req: 
```json
{
    "jsonrpc": "2.0",
    "method": "pathfinder_version",
    "params": [],
    "id": 0
}
```
Res:
```json
{
    "jsonrpc": "2.0",
    "error": {
        "code": -32601,
        "message": "Method not found"
    },
    "id": 0
}
```

**Comment by Mirko-von-Leipzig** [comment]
> I also tried this endpoint unusefully v.0.6.6 Req:
> 
> ```json
> {
>     "jsonrpc": "2.0",
>     "method": "pathfinder_version",
>     "params": [],
>     "id": 0
> }
> ```
> 
> Res:
> 
> ```json
> {
>     "jsonrpc": "2.0",
>     "error": {
>         "code": -32601,
>         "message": "Method not found"
>     },
>     "id": 0
> }
> ```

Are you trying on your own node, or on an api service like infura? If an api service, then its likely they don't expose it.

If its with your local node, it seems its only available on the `/rpc/pathfinder/v0.1/` endpoint.

The following curl command works for my local node:
```
curl http://127.0.0.1:9545/rpc/pathfinder/v0.1 \
 -s -X POST -H 'Content-Type: application/json' \
-d '{"jsonrpc":"2.0","id":"0","method":"pathfinder_version"}'
```
Outputs:
```json
{"jsonrpc":"2.0","result":"v0.6.6","id":"0"}
```

**Comment by tabaktoni** [comment]
> /rpc/pathfinder/v0.1/

Yea issue was the endpoint, it's working with .../rpc/pathfinder/v0.1/ 
Thx.
